src/main/webapp/js/user-page.js

```diff
@@ -219,10 +219,12 @@ function configureFollowButton(user, userLoggedInId) {
  * @param {object} user The user whose profile is being edited.
  */
 function showProfileEditingForm(user) {
+  const profileEditForms = document.getElementById('profile-edit-forms');
+  const profileInfoForm = document.getElementById('profile-form');
+  profileInfoForm.action += '/' + user.id;
+  profileEditForms.hidden = false;
```

Comment:
```suggestion
  const profileEditForms = document.getElementById('profile-edit-forms');
  profileEditForms.hidden = false;
  const profileInfoForm = document.getElementById('profile-form');
  profileInfoForm.action += '/' + user.id;
```

Reply:
May I know why it is better to do in this order?